extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/Tools.stack/Inspect.pulldown/Model Checker.pushbutton/script.py

```diff
@@ -475,18 +498,20 @@ def accents2ascii(s):
     worksetKind = str(worksetTable.GetWorkset(worksetId).Kind)
     if worksetKind == "UserWorkset":
         worksetNameAcc = worksetTable.GetWorkset(worksetId).Name
-        # nonaccented version for chart
-        worksetName = accents2ascii(worksetNameAcc)
+        # treating accents
+        worksetName = worksetNameAcc
         if worksetName not in worksetNames:
             worksetNames.append(worksetName)
         graphWorksetsData.append(worksetName)
 # print worksetNames
 # sorting results in chart legend
 worksetNames.sort()
+
 worksetsSet=[]
 for i in worksetNames:
     count=graphWorksetsData.count(i)        
     worksetsSet.append(count)
+worksetNames = [x.encode('utf8') for x in worksetNames]
```

Comment:
I added encoding also for worksets, if there was an accent in Workset name it caused an error